modules/reference/pages/admin/iss-target-setup.adoc

```diff
@@ -1,5 +1,5 @@
-[[ref-admin-iss-slave]]
-= ISS Slave Setup
+[[ref-admin-iss-target]]
+= ISS target Setup
 
 The menu:Admin[ISS Configuration > Slave Setup] section is used to configure an inter-server synchronization (ISS) slave.
```

Comment:
I am not sure its a great idea to change the naming from master/slave to source/target. The UI menus will confuse users. I know  these  names are not popular but we should stick with the naming conventions of the product.